[typescript-angular] Cannot compile when Swagger file has no definitions block

## Description

Given a Swagger definition without any model definitions, generated **typescript-angular** module can't be compiled:
``` ERROR in [...]/index.ts(2,15): error TS2306: File '[...]/model/models.ts' is not a module.```

## Swagger-codegen version
2.4.11-SNAPSHOT (b22ed93)

## Command line used
```bash
java -jar modules/swagger-codegen-cli/target/swagger-codegen-cli.jar generate \
    -i hello-world.yaml \
    -l typescript-angular \
    -o src/hello-world-consumer
```

## Swagger declaration file content or url
```yaml
swagger: '2.0'
info:
  version: 1.0.0
  title: Hello World API
schemes:
  - https
host: localhost
basePath: /api
paths:
  /hello-world:
    get:
      summary: Hello World
      responses:
        '200':
          description: Successful invokation
```


## Steps to reproduce
I created an example repository to reproduce the issue: https://github.com/yzhan94/swagger-codegen-angular-issue
 - Run `npm install` and `npm start` (node version: `10.17.0`).

## Suggest a fix

I believe this line should be wrapped in a conditional, so that it is omitted when no models are defined.

https://github.com/swagger-api/swagger-codegen/blob/b22ed93b4797a5c511c6a0e80d101ab33efd6538/modules/swagger-codegen/src/main/resources/typescript-angular/index.mustache#L2

--- a/modules/swagger-codegen/src/main/resources/typescript-angular/index.mustache
+++ b/modules/swagger-codegen/src/main/resources/typescript-angular/index.mustache
@@ -1,5 +1,9 @@
 export * from './api/api';
+{{#models}}
+{{#-first}}
 export * from './model/models';
+{{/-first}}
+{{/models}}
 export * from './variables';
 export * from './configuration';
 export * from './api.module';

